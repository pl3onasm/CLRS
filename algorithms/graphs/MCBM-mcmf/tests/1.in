MIN

3 7 30.4
1 4 10
1 6 30
6 3 20
2 4 5
5 2 20.3
2 6 10
7 2 60
7 1 40.5
4 3 18
3 5 20.1
1 5 23

