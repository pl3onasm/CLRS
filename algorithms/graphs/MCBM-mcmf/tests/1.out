
--------------------
 Input Network N
 Weighted
 Vertices: 7
 Edges: 12
--------------------
  1[3]: 4(1,10), 6(1,30), 5(1,23)
  2[2]: 4(1,5), 6(1,10)
  3[2]: 7(1,30.40), 5(1,20.10)
  4[1]: 3(1,18)
  5[1]: 2(1,20.30)
  6[1]: 3(1,20)
  7[2]: 2(1,60), 1(1,40.50)
--------------------


--------------------
 New Network N'
 Weighted
 Vertices: 9
 Edges: 19
--------------------
  1[1]: SINK(1,0)
  2[1]: SINK(1,0)
  3[1]: SINK(1,0)
  4[3]: 1(1,10), 2(1,5), 3(1,18)
  5[3]: 2(1,20.30), 3(1,20.10), 1(1,23)
  6[3]: 1(1,30), 3(1,20), 2(1,10)
  7[3]: 3(1,30.40), 2(1,60), 1(1,40.50)
  SINK[0]
  SOURCE[4]: 4(1,0), 5(1,0), 6(1,0), 7(1,0)
--------------------

--------------------
 Matching
 Cardinality: 3
 Min cost: 40.10
--------------------
  5 -- 3:   20.10
  6 -- 2:   10
  4 -- 1:   10
--------------------